client/src/state/Analytics/AnalyticsActions.ts

```diff
@@ -20,12 +20,7 @@ export function importAnalytics(): any {
         track('Analytics', 'import');
         try {
             const { users, interactions } = await API.importAnalytics();
-            track(
-                'Analytics',
-                'import',
-                'content-types',
-                `${interactions.map((i) => i.name)}`
-            );
+            track('Analytics', 'import', `${interactions.map((i) => i.name)}`);
```

Comment:
Shouldn't the array be converted a a string? The array doesn't show up in Matomo in the old version.

But is it really useful to have a list of interactions that is arbitrarily ordered and of varying length? I don't think we'll gain much from it. I think more interesting would be to know how many interactions typically must be displayed and to individually know which interactions are displayed to decide where to improve reports first.

Reply:
It is converted into a sting through the `${}`
You are right. I will change it to display the length of the array.